src/es.json

```diff
@@ -1110,8 +1110,8 @@
             "taskType": "Tipo de tarea",
             "unassignedTask": "Tarea no asignada",
             "group": "Grupo",
-            "indent": "Sangrar",
-            "outdent": "Outdent",
+            "indent": "Sangría",
+            "outdent": "Anular sangría",
```

Comment:
Use this "Eliminar sangría" which means remove the indent

Reply:
Changed